src/pretix/presale/checkoutflow.py

```diff
@@ -423,6 +427,19 @@ def all_optional(self):
                 return True
         return False
 
+    def get_contact_overrides(self):
+        overrides = contact_form_fields_overrides.send_chained(
+            self.request.event,
+            'contact_form_fields_overrides',
+            request=self.request
+        )
+
+        self.contact_overrides = True
```

Comment:
Can we name this ``contact_overrides_computed`` or something like that to make clear it doesn't say whether there are overrides, but only whether they've been handled already?